plugins/available/proxy.plugin.bash

```diff
@@ -13,53 +13,50 @@ disable-proxy ()
 	unset ALL_PROXY
 	unset no_proxy
 	unset NO_PROXY
-	echo "Disabled proxy environment variables"
+	_log_notice "Disabled proxy environment variables"
```

Comment:
Hmm ... Isn't this a user-facing message?  Will `_log_notice` show the message at user-invocation time?  Or do the loggers hide messages when not in `doctor` mode ?

Reply:
That's why I switched it back to draft after so happily marking it ready! 😞 

I accidentally cherry-picked this off a branch with unrelated changes to logging so I have to do it over again. 

But yes, currently all log messages are hidden by default.